Include the basic migration types [ci skip] (#39127)
rails/rails: guides/source/active_record_migrations.md

> You can append as many column name/type pairs as you want.
> 
> The native migration types are: :primary_key, :string, :text, :integer, :bigint, :float, :decimal, :numeric, :datetime, :time, :date, :binary, :boolean.

Reviewer: I think these would be better described as "column types" rather than "migration types."  Also, this reads well as part of the previous paragraph.

```suggestion
You can append as many column name/type pairs as you want. Common column types include: `bigint`, `binary`, `boolean`, `date`, `datetime`, `decimal`, `float`, `integer`, `numeric`, `primary_key`, `string`, `text`, and `time`.
```
